packages/discovery-provider/plugins/notifications/src/processNotifications/mappers/usdcPurchaseBuyer.ts

```diff
@@ -171,9 +193,9 @@ export class USDCPurchaseBuyer extends BaseNotification<USDCPurchaseBuyerRow> {
       html: email({
         purchaserName: purchaserUsername,
         artistName: sellerUsername,
-        trackTitle: purchasedTrackName,
-        trackLink: `${getHostname()}/${sellerHandle}/${track.slug}`,
-        trackImage: `${getContentNode()}/content/${track.cover_art_sizes}/480x480.jpg`,
+        contentTitle: purchasedContentName,
+        contentLink: `${getHostname()}/${sellerHandle}/${slug}`,
```

Comment:
Just to make sure, slug does include the route prefix? (e.g. `/album/my-album/`)

Reply:
wow excellent catch, it does not.

Reply:
Uh oh. Do the track ones work currently then? It doesn't look like it's getting appended anywhere to those...

Reply:
track links don't need the `/track`, see these:
https://audius.co/disclosure/album/alchemy-1246261550
https://audius.co/disclosure/looking-for-love-4